src/engraving/libmscore/tuplet.cpp

```diff
@@ -1431,4 +1431,16 @@ void Tuplet::addMissingElements()
              missingElementsDuration.numerator(), missingElementsDuration.denominator());
     }
 }
+
+int Tuplet::computeTupletDenominator(int numerator, Fraction totalDuration)
+{
+    Fraction noteDuration = totalDuration / numerator;
```

Comment:
It would be better to make sure that we don't divide by zero here

Reply:
right, good point